tests/src/main/java/com/hazelcast/stabilizer/tests/map/MapHeapHogWarmupTest.java

```diff
@@ -0,0 +1,136 @@
+import static com.hazelcast.stabilizer.tests.helpers.HazelcastTestUtils.nextKeyOwnedBy;
+import static com.hazelcast.stabilizer.utils.CommonUtils.sleepSeconds;
+import static java.lang.String.format;
+
+/**
+ * Fills up the cluster to a defined heap usage factor during warmup phase.
+ * <p/>
+ * This tests intentionally has an empty run phase.
+ */
+public class MapHeapHogWarmupTest {
+
+    private static final int MIN_CLUSTER_SIZE = 3;
+    private static final long APPROX_ENTRY_BYTES_SIZE = 238;
+    private static final ILogger LOGGER = Logger.getLogger(MapHeapHogWarmupTest.class);
+
+    // properties
+    public String basename = this.getClass().getSimpleName();
+    public int threadCount = 10;
+    public int ttlHours = 24;
+    public double approxHeapUsageFactor = 0.9;
+
+    private HazelcastInstance targetInstance;
+    private ThreadSpawner threadSpawner;
+    private IMap<Long, Long> map;
+
+    private long maxEntriesPerThread;
+
+    @Setup
+    public void setup(TestContext testContext) {
+        targetInstance = testContext.getTargetInstance();
+        threadSpawner = new ThreadSpawner(testContext.getTestId());
+        map = targetInstance.getMap(basename);
+
+        // calculate how many entries we have to insert per member and thread
+        long free = Runtime.getRuntime().freeMemory();
+        long total = Runtime.getRuntime().totalMemory();
+        long max = Runtime.getRuntime().maxMemory();
+
+        long used = total - free;
+        long totalFree = max - used;
+
+        long maxLocalEntries = (long) ((totalFree / APPROX_ENTRY_BYTES_SIZE) * approxHeapUsageFactor);
+        maxEntriesPerThread = (long) ((double) maxLocalEntries / (double) threadCount);
+    }
+
+    @Warmup()
+    public void warmup() {
+        if (isClient(targetInstance)) {
+            return;
+        }
+
+        // wait for the cluster to build up
+        while (targetInstance.getCluster().getMembers().size() != MIN_CLUSTER_SIZE) {
+            LOGGER.info(basename + " waiting to hit cluster size " + MIN_CLUSTER_SIZE);
+            sleepSeconds(1);
+        }
+
+        // fill the cluster as fast as possible with data
+        long startKey = 0;
+        for (int i = 0; i < threadCount; i++) {
+            threadSpawner.spawn(new FillMapWorker(startKey));
+            startKey += maxEntriesPerThread;
+        }
+        threadSpawner.awaitCompletion();
+
+        StringBuilder sb = new StringBuilder("After warmup of ").append(basename).append("\nmap size = ").append(map.size());
+        addMemoryStatistics(sb);
+        LOGGER.info(sb.toString());
+    }
+
+    @Verify()
+    public void verify() {
+        if (isClient(targetInstance)) {
+            return;
+        }
+
+        StringBuilder sb = new StringBuilder("Local verify of ").append(basename).append("\nmap size = ").append(map.size());
+        addMemoryStatistics(sb);
+        LOGGER.info(sb.toString());
+    }
+
+    @Run
+    public void run() {
```

Comment:
Again, I can understand the intention. See this quote from Danny

> The purpose of this test, was to create a cluster where each member was using a high proportion of heap space. then terminate one of the members. this leads to a cluster melt down and all members crash. I was not interested in, printing out messages a iteration, or measurint any kind of performance of any thing, its purely for, this melt down senario. so the best thing for this test would be to remove the timeout in the warmup / setup phase, and leave the run method empty

Maybe we should move this to distinct package for `non-standard` tests. I pushed it into the `misc` package in the meantime, but this didn't seem right, too.